addons/binding/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/jsons/JsonEqualizer.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2010-2019 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.amazonechocontrol.internal.jsons;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link JsonActivity} encapsulate the GSON data of the get equalizer command
+ *
+ * @author Michael Geramb - Initial contribution
+ */
+@NonNullByDefault
+public class JsonEqualizer {
```

Comment:
Does it make sense to implement the `Cloneable` interface?

Reply:
Sorry, I don't understand the question. Why should it make sense to implement this interface? I do not require a clone functionallity

Reply:
Maybe I am wrong but what is the purpose of this class? Especially the `createClone` method?

Reply:
I checked the code, but this is a special case and it's used only with the real type. I assume, the Cloneable interface does only make sense, if the implementation is used with an untyped object . So I do not see an advantage here.

Reply:
Alright. Thanks. Let's leave it like it is.